### PR TITLE
0.96.0 

### PR DESCRIPTION
**Pull request by tgoyne**
Closes #2449, #2458, #2475, #2476.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -35,6 +35,10 @@ x.x.x Release notes (yyyy-MM-dd)
 
 * Enable bitcode for iOS and watchOS frameworks.
 * Build libraries with Xcode 7 final rather than the GM.
+* Fix assertion failure when inserting NSData between 8MB and 16MB in size.
```
Comment (jpsim) [inline CHANGELOG.md]: This is missing a number of entries.

**Comment by tgoyne** [comment]
Test failures are the usual spurious ones plus the ios-devices test not running due to that machine being offline, so I think this is ready to go.

**Comment by jpsim** [comment]
Agreed, this looks ready to be released manually. :+1:

**Comment by jpsim** [comment]
Closing as this is now released: https://github.com/realm/realm-cocoa/releases/tag/v0.96.0-beta

**Pull request merged**